dowhy/timeseries/timeseries.py

```diff
@@ -0,0 +1,36 @@
+import networkx as nx
```

Comment:
is there is a better name for this file? Is it constructing the timeseries dataset?